Function.cs

```diff
@@ -32,6 +32,7 @@ public class Function
             "Arsenic", 
             "As Dark As My Soul Black", 
             "Azure", 
+            "Baby blue",
```

Comment:
Someone else beat you to this one, it's down toward the bottom.
```suggestion

```

Reply:
Done!